src/System.ValueTuple/tests/project.json

```diff
@@ -0,0 +1,133 @@
+{
+  "dependencies": {
+    /*"System.Runtime": "4.4.0-beta-24829-02",
```

Comment:
They do, but this is temporary. Last commit is "work in progress" (I needed to sync from home to office).